java/src/main/java/ai/rapids/cudf/ast/Literal.java

```diff
@@ -200,6 +200,15 @@ public static Literal ofDurationFromLong(DType type, Long value) {
     return ofDurationFromLong(type, value.longValue());
   }
 
+  /** Construct a string literal with the specified value or null. */
+  public static Literal ofString(String value) {
+    byte[] stringBytes = value.getBytes();
```

Comment:
This doesn't handle `value` being null.  Need to do something like this:
```suggestion
    if (value == null) {
      return ofNull(DType.STRING)
    }
    byte[] stringBytes = value.getBytes();
```